### PR TITLE
Update to guava 33.0.0.jre

### PR DESCRIPTION
**Pull request by merks**

### CODE REVIEW

**Comment by merks** [comment]
@cdietrich @jonahgraham 

Could someone approve the build?

This change is analogous to the following:

https://github.com/eclipse-lsp4j/lsp4j/commit/523168fb44987087e638ecbfa6640c9f963a774e

File: releng/releng-target/lsp4j.target.target [hunk 1]
```diff
@@ -6,8 +6,8 @@
 			<repository location="https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2023-09"/>
 			<unit id="com.google.gson" version="2.10.1"/>
 			<unit id="com.google.gson.source" version="2.10.1"/>
-			<unit id="com.google.guava" version="32.1.2.jre"/>
```
Comment (cdietrich) [inline releng/releng-target/lsp4j.target.target:9]: i assume you need to split this up into two orbit locations

Comment (cdietrich) [reply releng/releng-target/lsp4j.target.target:9]: https://github.com/eclipse-lsp4j/lsp4j/compare/cd_widenGuava

**Comment by cdietrich** [comment]
plus one. then we can release Xtext with the 0.21.2 dependency in case there wont be a release in lsp4j for 2024-03 and lsp4j can then consume the Xtext 2.34 release

File: releng/releng-target/lsp4j.target.target (hunk 1 above)
Comment (jonahgraham) [reply releng/releng-target/lsp4j.target.target:9]: Why do you think that the orbit should be split up here? I want to make sure I don't mess things up in 0.21.2 that I am working on.

Comment (merks) [reply releng/releng-target/lsp4j.target.target:9]: For sure the new version of guava needs 2024-03 to be added...

Comment (jonahgraham) [reply releng/releng-target/lsp4j.target.target:9]: I think I answered the question. It is about the renaming of the jakarta.websocket bundles?

Comment (merks) [reply releng/releng-target/lsp4j.target.target:9]: Yes, I think these 

image

Comment (jonahgraham) [reply releng/releng-target/lsp4j.target.target:9]: Thanks Ed. I think something is going wrong here as we don't use the older jakarta.websocket, but instead jakarta.websocket-api in the code, so I will look at that.

**Pull request closed without merging**